eudemo/eudemo/reactor.py

```diff
@@ -345,10 +345,15 @@ def _read_json(file_path: str) -> Dict:
         params, build_config.get("RadiationShield", {}), reactor.cryostat.xz_boundary()
     )
 
+    reactor.show_cad(
+        n_sectors=3,
+        # with_components=[
+        #     reactor.pf_coils,
+        #     reactor.plasma,
+        # ],
```

Comment:
Ditto

Reply:
Left that in there so show how to use it to display only those comps.

Don't mind removing it.